src/razor/src/razorLanguageServerClient.ts

```diff
@@ -268,10 +268,8 @@ export class RazorLanguageServerClient implements vscode.Disposable {
             args.push('--SingleServerCompletionSupport');
             args.push('true');
 
-            if (options.forceRuntimeCodeGeneration) {
-                args.push('--ForceRuntimeCodeGeneration');
-                args.push('true');
-            }
+            args.push('--ForceRuntimeCodeGeneration');
```

Comment:
two questions
1.  Does this need to be backported to prerelease?
2.  If we are backporting to prerelease, are we OK with the release going out next week with fuse on?

Reply:
1. yes 
2. no

Assuming by doing 1 we prevent fuse from default on in vs code. Will see what the best fix is moving forward. We could probably get closer to VS behavior with https://github.com/dotnet/razor/issues/9150 and avoid part of this problem (although the missing source mapping should still be added, which will be a separate issue)

Reply:
My apologies, I didn't read the change closely enough.  I see now its switching back to the option.  LGTM

Reply:
Also realized we don't have the backport bot here yet - PR for that here - https://github.com/dotnet/vscode-csharp/pull/8021